src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -321,6 +354,7 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             }
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP doesn't expose channel binding information")]
```

Comment:
This is more about a missing feature in UAP.  Channel binding information is used but the WinRT API doesn't expose it up-stack. Hence, we can't expose it in the .NET layer either.  So, we simply don't run the test on UAP.  If we were to write a test to "pin" this behavior, the test would simply be validating a null byte array for channel binding information is returned.  Is that what you are suggesting I add?